### PR TITLE
py-absl: Update to version 0.11.0

### PR DESCRIPTION
**Pull request by essandess**
py-absl: Update to version 0.11.0

* Update to version 0.11.0
* Remove python version 35

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.7 19H2
Xcode 12.1 12A7403

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-absl/Portfile [hunk 1]
```diff
@@ -18,16 +18,15 @@ homepage            https://github.com/abseil/abseil-py
...
 
-checksums           rmd160  e2fa96fb03bedd2a4d5e4de09969bf116ab7f593 \
-                    sha256  d9129186431e150d7fe455f1cb1ecbb92bb5dba9da9bc3ef7b012d98c4db2526 \
-                    size    103181
+checksums           rmd160  64942f2658e1951133fe14ab388f375399f23d7f \
+                    sha256  b20f504a7871a580be5268a18fbad48af4203df5d33dbc9272426cb806245a45 \
+                    size    110054
 
-python.versions     27 35 36 37 38
+python.versions     37 38
```
Comment (reneeotten) [inline python/py-absl/Portfile:25]: you cannot remove subports on which other ports still depend:
```
>>> port echo depends:py27-absl ; port echo depends:py35-absl ; port echo depends:py36-absl
py27-tensorboard1
py27-tensorflow1
py36-tensorboard1
py36-tensorflow1
```

File: python/py-absl/Portfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-absl
```
Comment (reneeotten) [inline python/py-absl/Portfile:6]: please add `python.rootname  absl-py` here and then remove the `master_sites`, `distname`, and `livecheck.name` fields below

File: python/py-absl/Portfile [hunk 3]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
...
-version             0.8.1
+version             0.10.0
```
Comment (reneeotten) [inline python/py-absl/Portfile:7]: considering updating to the latest version, currently 0.11.0 (I know it was the latest when you submitted this PR...)

File: python/py-absl/Portfile (hunk 1 above)
Comment (reneeotten) [reply python/py-absl/Portfile:25]: also when restoring the PY27 subport, please note that the `py-enum34` dependency is missing so please add that below as:
```
if {${python.version} == 27} {
    depends_lib-append port:py${python.version}-enum34
}
```

File: python/py-absl/Portfile [hunk 4]
```diff
@@ -40,5 +39,5 @@ if {${name} ne ${subport}} {
         file attributes ${worksrcpath}/absl_py.egg-info/PKG-INFO -permissions a+r
```
Comment (reneeotten) [inline python/py-absl/Portfile:39]: please verify whether this is still needed, hopefully upstream has fixed the permissions in the newer release and, if so, you can remove this block

**Review by reneeotten — CHANGES_REQUESTED** [review]
a few changes and then we should be able to merge quickly

File: python/py-absl/Portfile (hunk 1 above)
Comment (essandess) [reply python/py-absl/Portfile]: Thanks done.

File: python/py-absl/Portfile (hunk 3 above)
Comment (essandess) [reply python/py-absl/Portfile]: Thanks done.

File: python/py-absl/Portfile (hunk 4 above)
Comment (essandess) [reply python/py-absl/Portfile]: No longer needed. Thanks done.

**Comment by essandess** [comment]
Thanks for all the corrections and comments @reneeotten.

**Review by reneeotten — APPROVED** [review]

**Pull request merged**